packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -136,6 +142,47 @@ function ExploreTemplates() {
     [handleCreateStoryFromTemplate]
   );
 
+  const handleDetailsToggle = useCallback(
+    (id) => {
+      setIsDetailsViewOpen((prevIsOpen) => {
+        const newIsOpen = !prevIsOpen;
+        // should we add a tracking event like so?
+        trackEvent('details_view_toggled', {
+          status: newIsOpen ? 'open' : 'closed',
+        });
```

Comment:
We should continue to use `trackScreenView()` like we currently do in `InterfaceSkeleton`